### ISSUE TITLE
My training perplexity is too large and fixed

### ISSUE THREAD

**Issue by he1ght** [OpenNMT/OpenNMT-py#676]
Hi, I'm a student studying OpenNMT-py.
I have a question.

My environment is 
pytorch 0.3.0
ubuntu 16.04

I used code with these options
`python train.py -data ../NMT_data/wmt/fr-en/demo -save_model model/4L1000-fr-en/demo -visualization -layers 4 -rnn_size 1000 -gpuid 0 -optim adam`
and training dataset from WMT15 fr-en 1.5M

but I got too large perplexity and too small accuracy than SGD optimizer.
`Epoch  1, 8900/1275; acc:   1.61; ppl: 26881171418161356094253400435962903554686976.00; 3395 src tok/s; 3169 tgt tok/s;   4270 s elapsed
Epoch  1, 8950/1275; acc:   1.60; ppl: 26881171418161356094253400435962903554686976.00; 2915 src tok/s; 3130 tgt tok/s;   4295 s elapsed
Epoch  1, 9000/1275; acc:   1.46; ppl: 26881171418161356094253400435962903554686976.00; 3237 src tok/s; 3102 tgt tok/s;   4319 s elapsed
Epoch  1, 9050/1275; acc:   1.51; ppl: 26881171418161356094253400435962903554686976.00; 3006 src tok/s; 3128 tgt tok/s;   4343 s elapsed
Epoch  1, 9100/1275; acc:   1.73; ppl: 26881171418161356094253400435962903554686976.00; 3209 src tok/s; 3210 tgt tok/s;   4367 s elapsed
Epoch  1, 9150/1275; acc:   1.61; ppl: 26881171418161356094253400435962903554686976.00; 3283 src tok/s; 3198 tgt tok/s;   4390 s elapsed
Epoch  1, 9200/1275; acc:   1.57; ppl: 26881171418161356094253400435962903554686976.00; 2913 src tok/s; 3113 tgt tok/s;   4415 s elapsed
Epoch  1, 9250/1275; acc:   1.63; ppl: 26881171418161356094253400435962903554686976.00; 2851 src tok/s; 3066 tgt tok/s;   4441 s elapsed`

Is this wrong result?
Did I give wrong options?

**Comment by sebastianGehrmann** [comment]
With adam you need to use a way smaller learning rate such as `-learning_rate 0.003`.

**Comment by he1ght** [comment]
Thanks. It works well!

I'm so sorry that asked before searching previous issues.
Thanks for quick response
